core/debug/src/core_armv7m/mpu_armv7m/debug_armv7m.c

```diff
@@ -36,22 +36,22 @@ void debug_mpu_config(void)
     char dim[][3] = {"B ", "KB", "MB", "GB"};
     int i;
 
-    dprintf("* MPU CONFIGURATION\n\r");
+    dprintf("* MPU CONFIGURATION\n");
 
     /* CTRL */
     ctrl = MPU->CTRL;
-    dprintf("\n\r");
-    dprintf("  Background region %s\n\r", ctrl & MPU_CTRL_PRIVDEFENA_Msk ?
+    default_putc('\n');
```

Comment:
why are we calling `default_putc` directly? this way it will be harder for us to redirect/disaable logs.
I believe we should use the same macro/function for all prints

Reply:
Fixed to use `dprintf` instead of `default_putc`